commands/removeCardCommand.js

```diff
@@ -1,36 +1,69 @@
 import db from "../db.js";
 
-const removeCardCommand = (message) => {
+const removeCardCommand = async (message) => {
   const cardToRemove = message.content.split(" ").slice(1).join(" ");
-  const ref = db.ref("users");
-  ref.once("value", (snapshot) => {
-    const userData = snapshot.val();
-    const userID = message.author.id;
-    const userInventory = userData[userID].inventory;
-
-    const cardIDToRemove = Object.keys(userInventory).find(
-      (key) =>
-        userInventory[key].name.toLowerCase() === cardToRemove.toLowerCase()
+  const ref = db.ref(`users/${message.author.id}`);
+  const snapshot = await ref.once("value");
+  const userData = snapshot.val();
```

Comment:
this is purely a stylistic choice, but it's actually possible to condense this to:
```jsx
const userData = (await ref.once("value")).val();
```

But people are very divided on whether this is better or not. I'd make the change but perhaps most would stick with what you have to avoid overloading a single line